package.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I'm unfamiliar with npm package management. Do you need to put your web address or nbnovnc's for homepage,  bugs, repo, etc.?

Reply:
It's not really my domain either... I do not think it's mandatory.